ci/build_wheel.sh

```diff
@@ -6,9 +6,20 @@ set -euo pipefail
 rapids-logger "Install CUDA Toolkit"
 source "$(dirname "$0")/install_latest_cuda_toolkit.sh"
 
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+
+# This is the version of the suffix with a preceding hyphen. It's used
+# everywhere except in the final wheel name.
+PACKAGE_CUDA_SUFFIX="-${RAPIDS_PY_CUDA_SUFFIX}"
+
+# Patch project metadata files to include the CUDA version suffix and version override.
+pyproject_file="${package_dir}/pyproject.toml"
+
+sed -i "s/^name = \"${package_name}\"/name = \"${package_name}${PACKAGE_CUDA_SUFFIX}\"/g" ${pyproject_file}
+
 rapids-logger "Build wheel"
-mkdir -p ./wheel-build
-pip wheel . --wheel-dir=./wheel-build -vvv
+mkdir -p ./dist
```

Comment:
I renamed this to `dist` to align with other RAPIDS packages that use this directory name for wheels.